components/inputs/input-text.js

```diff
@@ -41,6 +41,10 @@ class InputText extends RtlMixin(LitElement) {
 			 * Disables the input
 			 */
 			disabled: { type: Boolean, reflect: true },
+			/**
```

Comment:
I added `hideInvalidIcon` because in the validation designs, input-date has the calendar icon replaced with the alert icon when invalid input occurs.